ts/hooks/useHeaderSecondLevel.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from "react";
+import { HeaderSecondLevel } from "@pagopa/io-app-design-system";
+import { useLayoutEffect } from "react";
+import { useNavigation } from "@react-navigation/native";
+import { FAQsCategoriesType } from "../utils/faq";
+import I18n from "../i18n";
+import {
+  ContextualHelpProps,
+  ContextualHelpPropsMarkdown
+} from "../components/screens/BaseScreenComponent";
+import { useStartSupportRequest } from "./useStartSupportRequest";
+
+type CommonProps = {
+  title: string;
+  backAccessibilityLabel?: string;
+  goBack?: () => void;
+};
+
+type PropsWithSupport = CommonProps & {
+  supportRequest: true;
+  faqCategories?: ReadonlyArray<FAQsCategoriesType>;
+  contextualHelp?: ContextualHelpProps;
+  contextualHelpMarkdown?: ContextualHelpPropsMarkdown;
+};
+
+type PropsWithoutSupport = CommonProps & {
+  supportRequest?: false;
+  faqCategories?: never;
+  contextualHelp?: never;
+  contextualHelpMarkdown?: never;
+};
+
+export type HeaderSecondLevelHookProps = PropsWithSupport | PropsWithoutSupport;
+
+type HeaderProps = React.ComponentProps<typeof HeaderSecondLevel>;
+
+/**
+ * A hook to set the header of a second level screen with useLayoutEffect hook
+ * this hook only handles 2 basic cases of the header, with or without support request
+ * rendering only the singleAction scenario of the header, in case of multiple actions it is needed to develop a custom header.
+ * @param props
+ */
+export const useHeaderSecondLevel = ({
+  title,
+  backAccessibilityLabel,
+  contextualHelp,
+  contextualHelpMarkdown,
+  faqCategories,
+  goBack,
+  supportRequest
+}: HeaderSecondLevelHookProps) => {
```

Comment:
What do you think about implementing the ability to add a second action via the props of this hook?
Maybe an optional `ActionProp` to pass in case there is the need to show an additional action.

Reply:
nice suggestion, at the beginning I was trying not to complicate the hook code, but it's indeed useful

[de3a189](https://github.com/pagopa/io-app/pull/5028/commits/de3a1895c2e9f2c827dc1bdb1a6f3cca152d4d26) implements it